src/include/mkdirp/mkdirp.c

```diff
@@ -0,0 +1,82 @@
+#include <unistd.h>
```

Comment:
We should have a full copy of the MIT license from the mkdirp library in here, since we are copying and redistributing this library.